include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -541,7 +557,7 @@ struct OrtCustomOp {
   uint32_t version;  // Initialize to ORT_API_VERSION
 
   // This callback creates the kernel, which is a user defined parameter that is passed to the Kernel* callbacks below.
-  void(ORT_API_CALL* CreateKernel)(_In_ struct OrtCustomOp* op, _In_ OrtKernelInfo* info, _Out_ void** op_kernel);
+  void(ORT_API_CALL* CreateKernel)(_In_ struct OrtCustomOp* op, _In_ const OrtCustomOpApi* api, _In_ OrtKernelInfo* info, _Out_ void** op_kernel);
```

Comment:
_In_ const OrtKernelInfo* info
or remove the _In_.

Reply:
I made it const and fixed up the downstream effects of it too.